defaults/defaults.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for defaults 1.0.3
+// Project: https://github.com/tmpvar/defaults/
+// Definitions by: Ibtihel CHNAB <https://github.com/IbtihelCHNAB/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function defaults(options: Object, defaultOptions: Object): Object;
```

Comment:
This could be `declare function defaults<T>(options: any, defaultOptions: T): T`, assuming there is a default for every option. If not, still prefer `any` to `Object`: `declare function defaults(options: any, defaultOptions: any): any;`

As a side note, partial types (https://github.com/Microsoft/TypeScript/issues/4889) would make this easier, but they're not available yet.